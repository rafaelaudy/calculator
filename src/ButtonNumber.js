import React from "react";

const ButtonNumber = ({ number }) => {
  return <button>{number}</button>;
};

export default ButtonNumber;
