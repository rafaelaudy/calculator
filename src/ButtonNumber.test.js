import React from "react";
import { shallow } from "enzyme";
import ButtonNumber from "./ButtonNumber";

it("Button is correctly loaded", () => {
  const buttonNumber = shallow(<ButtonNumber number={1} />);
  expect(buttonNumber).toMatchSnapshot();
});
