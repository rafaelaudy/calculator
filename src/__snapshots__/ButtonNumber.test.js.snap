// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button is correctly loaded 1`] = `
<button>
  1
</button>
`;
